### PR TITLE
stolostron/backlog#27160 ki id.k8s.io missing 311

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by zhiweiyin318** [comment]
thanks.
/lgtm

File: clusters/release_notes/known_issues.adoc [hunk 1]
```diff
@@ -33,6 +33,20 @@ Review the known issues for {product-title}. The following list contains known i
 [#cluster-lifecycle-issues]
 == Cluster lifecycle
 
+[#id-k8s-io-not-updated-311}
```
Comment (oafischer) [inline clusters/release_notes/known_issues.adoc]:
```suggestion
[#id-k8s-io-not-updated-311]
```
Small typo

File: clusters/release_notes/known_issues.adoc [hunk 2]
```diff
@@ -33,6 +33,20 @@ Review the known issues for {product-title}. The following list contains known i
...
+=== Value of the cluster claim _id.k8s.io_ on {ocp-short} version 3.11 is lost after upgrading to {product-title-short} 2.7
+//2.7:ACM3000
+
+After upgrading to {prodcuct-title-short} 2.7, the value of cluster claim `id.k8s.io` is missing for a managed cluster that is running on {ocp-short} 3.11. The value is stored in the `clusterclaims.cluster.open-cluster-management.io` file.
```
Comment (oafischer) [inline clusters/release_notes/known_issues.adoc]:
```suggestion
After upgrading to {product-title-short} 2.7, the value of cluster claim `id.k8s.io` is missing for a managed cluster that is running on {ocp-short} 3.11. The value is stored in the `clusterclaims.cluster.open-cluster-management.io` file.
```

File: clusters/release_notes/known_issues.adoc [hunk 3]
```diff
@@ -33,6 +33,20 @@ Review the known issues for {product-title}. The following list contains known i
...
+
+To cause the value to repopulate, manually delete the `id.k8s.io` setting by entering the following command:
+
+----
+oc delete clusterclaims.cluster.open-cluster-management.io  id.k8s.io
+----
+
+The clusterclaim is re-created automatically.
```
Comment (oafischer) [inline clusters/release_notes/known_issues.adoc]:
```suggestion
The cluster claim is re-created automatically.
```

Just for consistency

**Review by oafischer — APPROVED** [review]
lgtm, just a few typos

**Pull request merged**